hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
```

Comment:
> thinking about this, does it make sense to have an oadm registry is-up command or something in the CLI to facilitate this?

I like the idea!

> Actually, that being said, can we use oc status here? Will that correctly tell us when the service is "ready"?

We could, but we we'd be waiting perhaps unnecessarily long:

    # when the deployer is running regardless of registry pod running or ready
    ...
    svc/docker-registry - 172.30.186.199:5000
      dc/docker-registry deploys docker.io/openshift/origin-docker-registry:latest 
        deployment #1 running for 40 seconds - 1 pod
    ...
    # when the deployer is deleted
    ...
    svc/docker-registry - 172.30.186.199:5000
      dc/docker-registry deploys docker.io/openshift/origin-docker-registry:latest 
        deployment #1 deployed about a minute ago - 1 pod
    ...

Somewhen in between the endpoints are set and the deployer destroyed registry becomes ready. On my setup the deployer is deleted 2 seconds after the registry becomes ready. Moreover I'd prefer parsing of pod's status rather than status description which can easily change.

Let me try with the `jsonpath`.